runtime/decimal.c

```diff
@@ -107,6 +109,37 @@ TaggedPtrPanicCode finish(decQuad *dq, decContext *cx) {
     return result;
 }
 
+bool _bal_decimal_exact_eq(TaggedPtr tp1, TaggedPtr tp2) {
+    decQuad dc1;
```

Comment:
Should be d1, d2 for consistency with other functions.

Reply:
Fixed